frontend/src/less/components/searchbox.less

```diff
@@ -1,15 +1,44 @@
 #searchbox {
+    width: 563px;
+    max-width: 100%;
+    margin: 0 auto;
+    position: relative;
+    @media screen and (max-width: 800px) {
+        margin-top: 30px;
+    }
+    @media screen and (max-width: 600px) {
+        padding: 0 20px
+        width: 100%;
+    }
+
     span {
         padding-right: 20px;
         line-height: 52px;
     }
 
+    >span {
```

Comment:
please use a class instead of span, also merge with the definition above (span and >span) as they look to be connected to the same span